brooklyn-tosca-transformer/src/test/java/io/cloudsoft/tosca/a4c/Alien4CloudIntegrationTest.java

```diff
@@ -34,8 +35,13 @@
     protected ToscaTypePlanTransformer transformer;
     protected Uploader uploader;
 
+    protected Boolean origFeatureEnablement;
+    
     @BeforeMethod(alwaysRun = true)
     public void setUp() throws Exception {
+    	origFeatureEnablement = BrooklynFeatureEnablement.isEnabled(ToscaTypePlanTransformer.FEATURE_TOSCA_ENABLED);
+    	BrooklynFeatureEnablement.setDefault(ToscaTypePlanTransformer.FEATURE_TOSCA_ENABLED, true);
```

Comment:
I wondered about the context in which these tests would be run - is it only ever in a JVM that is running just the brooklyn-tosca tests, or could it be running anything else (thus not rolling back would impact other things). I suspect it will always be a separate JVM (e.g. if doing a release build of a downstream project, then we'd build each repo in its own JVM).

I'm going to merge as-is anyway. Rolling back shouldn't do any harm.